e2e/components/block-scroll-strategy/block-scroll-strategy.e2e.ts

```diff
@@ -1,13 +1,13 @@
 import {browser, Key, element, by} from 'protractor';
 import {screenshot} from '../../screenshot';
-import {getScrollPosition} from '../../util/query';
+import {getScrollPosition, asyncSpec} from '../../util/index';
 
 
 describe('scroll blocking', () => {
   beforeEach(() => browser.get('/block-scroll-strategy'));
   afterEach(() => clickOn('disable'));
 
-  it('should not be able to scroll programmatically along the x axis', async (done) => {
+  it('should not be able to scroll programmatically along the x axis', asyncSpec(async () => {
```

Comment:
Hmm looks like it. I was initially playing around with passing in the `done` function and calling it at the end, which ended up timing out for failures since I wasn't wrapping it in a try/catch and calling `done.fail`. I guess that was overriding the custom handling from `jasminewd`. I've removed the `asyncSpec` now and it behaves as expected.